tests/test_utils.py

```diff
@@ -529,6 +529,29 @@ def test_query_string_in_path(self):
         with pytest.raises(ValueError):
             client.simulate_get(path='/thing?x=1')
 
+    def test_simulate_json_body(self):
```

Comment:
Please use ``pytest.mark.parametrize`` to test a few different types for ``document`` (e.g., dict, list, int, string), just to ensure we don't ever have a regression related to this.